x-pack/plugins/observability_onboarding/public/components/shared/windows_install_step.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiLink, EuiText } from "@elastic/eui";
+import { i18n } from "@kbn/i18n";
+import React from "react";
+
+export function WindowsInstallStep() {
+  return (
+    <EuiFlexGroup direction="column">
+      <EuiFlexItem grow={false}>
+        <EuiText color="subdued">
+          <p>
+            {i18n.translate(
+              'xpack.observability_onboarding.windows.installStep.description',
+              { defaultMessage: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.' }
+            )}
+          </p>
+        </EuiText>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiButton
+          iconSide="right"
+          iconType="popout"
+          color="primary"
+          href="https://www.elastic.co/guide/en/fleet/current/install-standalone-elastic-agent.html"
```

Comment:
@mdbirnstiehl is this te right link?

Reply:
I think we could send them to the Stream a log file docs: https://www.elastic.co/guide/en/observability/current/logs-stream.html

I think this will work better as long as we are ok with them manually configuring their agent without integrations and they want to use a standalone agent.

Reply:
I just realized this is for the system logs and not the stream logs onboarding like in the example we looked at in Slack. Will this message be used for both? If this is just about installing the agent, we can use the link you put originally.

Reply:
I placed the component in both flows: system and custom. For custom I will pass the link you referred to above.
We don't have any specific for system onboarding though

Reply:
Makes sense. That is probably the best link we have at this point for installing the agent then. I wonder if it would make sense to also point to the System integration?

Reply:
I found https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-observability.html, would that be a good entry point for documentation? We can replace the generic about installing standalone agent with this one if that one is too generic

Reply:
Ah yes! that is probably a much better place for them to get started for the system logs.

Reply:
Addressed in [f89cdf0](https://github.com/elastic/kibana/pull/163735/commits/f89cdf0f26b8bd2d10274cb480f0ad7a1cc791a7).